recipe/bld.bat

```diff
@@ -0,0 +1,36 @@
+if "%ARCH%"=="32" (
+    set PLATFORM=Win32
+) else (
+    set PLATFORM=x64
+)
+
+set mpfr_root=%cd%
+cd ..
+mkdir mpir\dll\%PLATFORM%\Release
+mkdir mpir\lib\%PLATFORM%\Release
+
+REM copy mpir headers
+xcopy %LIBRARY_INC%\*.h mpir\lib\%PLATFORM%\Release\ /E
+xcopy %LIBRARY_INC%\*.h mpir\dll\%PLATFORM%\Release\ /E
+
+REM copy mpir libraries
+copy %LIBRARY_LIB%\mpir.dll mpir\dll\%PLATFORM%\Release\mpir.dll
+copy %LIBRARY_LIB%\mpir.lib mpir\dll\%PLATFORM%\Release\mpir.lib
+copy %LIBRARY_LIB%\mpir_static.lib mpir\lib\%PLATFORM%\Release\mpir.lib
```

Comment:
You are correct.  Here are the rules: https://msdn.microsoft.com/en-us/library/7d83bc18.aspx

With miniconda/anaconda, we put %PREFIX%, %SCRIPTS%, and %LIBRARY_BIN% on PATH.